### ISSUE TITLE
Selecting a different primary color on start.vaadin.com is broken

### ISSUE THREAD

**Issue by markhm** [vaadin/platform#2075]
**Describe the bug**
I just created a new project based on v20.0.0.alpha8 with a navy blue primary color, but I don't see that the buttons have the color I selected. I also confirmed this for 19.0.6. 

**To Reproduce**
1. Go to 'start.vaadin.com' and start a new project based on Fusion, Java 16 and pnpm
2. Select a different primary color, e.g. navy blue
3. Download and run the project
4. See that the selected color is nót applied when run locally

<img width="1498" alt="Screenshot 2021-05-01 at 08 56 35" src="https://user-images.githubusercontent.com/1430451/116774220-651bd200-aa5b-11eb-912a-b7d7a65ca355.png">

**Expected behavior**
The selected primary color should be applied.

**Desktop (please complete the following information):**
 - Vaadin platform version: Confirmed for `v20.0.0.alpha8` and `19.0.6`.

**Comment by Artur-** [comment]
Hi,

I tried to reproduce this but couldn't. Changing the primary color also changes it in the downloaded app. Are there some more steps needed to reproduce the problem?

**Comment by markhm** [comment]
Thank you for checking, @Artur-. 

I just started the application I downloaded again and noticed this strange behaviour on Brave.
https://user-images.githubusercontent.com/1430451/116787919-34618a00-aaa7-11eb-8616-706858647b33.mov

I now also checked with Chrome and Safari and got the correct colours, so it is definitely something with my Brave browser. I will remove all cache etc. tomorrow and see if that fixes it. So closing issue again.
